api/internal/core/store/store_mock.go

```diff
@@ -18,48 +18,184 @@ package store
 
 import (
 	"context"
+	"encoding/json"
+	"fmt"
+	"sort"
 
+	"github.com/apisix/manager-api/internal/core/entity"
+	"github.com/shiningrush/droplet/data"
 	"github.com/stretchr/testify/mock"
 )
 
-type MockInterface struct {
-	mock.Mock
+type Mock struct {
+	MockInterface
```

Comment:
It would be better to keep an empty line after the embed object.

Reply:
Updated